### ISSUE TITLE
Add documentation regarding configuration needed with multiple switches

### ISSUE THREAD

**Issue by tzumainn** [CCI-MOC/esi#353]

**Comment by joachimweyl** [comment]
@hakasapl or @naved001 is this documentation still needed?

**Comment by naved001** [comment]
@joachimweyl Yes, this should be documented. The description of this issue discusses a particular case of our environment which has now changed, but the trunking of VLANs between multiple switches is still something that needs to be done for a general ESI deployment.
